docs/install.md

```diff
@@ -86,6 +86,10 @@ To install Tekton Dashboard on an OpenShift cluster:
 
 Congratulations! You have successfully installed Tekton Dashboard on your OpenShift cluster.
 
+**Note for users installing Tekton Pipelines and Triggers with the OpenShift Pipelines operator:**
+
+Tekton Pipelines and Triggers will be installed in the `openshift-pipelines` namespace and the Dashboard won't be able to lookup versions there out of the box. You will need to edit the deployment and remove `--pipelines-namespace=tekton-pipelines` and `--triggers-namespace=tekton-pipelines` args to make everything work correctly.
+
```

Comment:
In the staged release yaml, these flags won't be present - so you could mention that in two ways.

1) in tekton/README.md, releasers need to know to remove these flags (short of doing it automatically)
2) that means the above is only applicable for developers right? Surely we should provide an overlay that uses those namespaces instead and we qualify that the openshift overlays are for the operator installs?

Reply:
Not sure if we've cut a release with support for these flags yet, maybe there's no staged release that have them.

Before the flags came in we looked either in all namespaces or had a fallback to look in the `openshift-pipelines` namespace if not found in the install namespace. I changed the lookup logic to allow using `RoleBinding` instead of `ClusterRoleBinding`.

Do you suggest that openshift manifests we release are for the OpenShift Pipelines operator install path ?

I'm going to add support for `triggers-namespace` and `pipelines-namespace` in the installer script.

Reply:
*Do you suggest that openshift manifests we release are for the OpenShift Pipelines operator install path ?"

Exactly this yeah as that's how it's always been and we can doc that. You're right there's no staged release yet with the flags and hence why I'd like for this issue to effectively result in that, even if I have to do a little manual work for the yaml in the short term